src/stripe-payment.js

```diff
@@ -0,0 +1,220 @@
+/**
+ * `stripe-payment`
+ * `Payment broker for stripe`
+ *
+ * @demo demo/index.html
+ * @customElement `stripe-payment`
```

Comment:
```suggestion
 * @element stripe-payment
```
let's use Rune's jsdoc tags for [`web-component-analyzer`](https://github.com/runem/web-component-analyzer)